x-pack/plugins/fleet/common/openapi/components/schemas/agent_policy.yaml

```diff
@@ -61,6 +61,10 @@ properties:
   is_protected:
     description: Indicates whether the agent policy has tamper protection enabled. Default false.
     type: boolean
+  overrides:
+    type: object
+    description: Overrides agent policy keys. Overriding inputs is not allowed.
```

Comment:
I am curious if you have copy suggestion to emphasis this should be used as a last resort fix and not in a "normal" use case

Reply:
cc @karenzone in case you have a perfect wording for this.

Reply:
@nchaulet, how about this:
> Override settings that are defined in the agent policy. Input settings cannot be overridden. The override option should be used only in unusual circumstances and not as a routine procedure.

I wasn't sure if `keys` is the right term so I used `settings`, but please switch if that's not right.

Also, since you're in the file, up at line 36 would you mind changing
`This field is present only when retrieving a single agent policy, or when retrieving a list of agent policy with the...` to
`This field is present only when retrieving a single agent policy, or when retrieving a list of agent policies with the...`